### PR TITLE
Added ability to run two models in parallel to keep performance of both models

### PR DESCRIPTION
**Pull request by praveenVnktsh**
image

Why??
- COCO dataset is massive, our dataset is not
- If we merge datasets, performance for our data will degrade due to long tail distribution.
- Thus we run two models in parallel just because we can!

### CODE REVIEW

**Comment by praveenVnktsh** [comment]
We sample only classes that we care about now. Unnecessary classes are rejected by post-processing script.
 Current class list with appropriate id mappings in order;
```yaml
class_list:
  - 'soda_can'
  - 'tissue_paper'
  - 'toothbrush'
  - 'tie'
  - 'cell phone'
  - 'banana'
  - 'apple'
  - 'orange'
  - 'bottle'
  - 'cup'
  - 'teddy_bear'
```

**Comment by praveenVnktsh** [comment]
@AbhinavGupta121 bump for review pls

File: src/manipulation/scripts/fsm.py
Comment (AbhinavGupta121) [inline src/manipulation/scripts/fsm.py:1]: Multiple labels for bananas!

Comment (AbhinavGupta121) [inline src/manipulation/scripts/fsm.py:1]: Again multiple labels for banana

Comment (AbhinavGupta121) [inline src/manipulation/scripts/fsm.py:1]: same

File: src/manipulation/scripts/manip_basic_control.py
Comment (AbhinavGupta121) [inline src/manipulation/scripts/manip_basic_control.py:1]: Why was this changed?

Comment (AbhinavGupta121) [inline src/manipulation/scripts/manip_basic_control.py:1]: Can we change id numbers and use string labels instead that map to them. Would be easier to know which is it is.

Comment (praveenVnktsh) [reply src/manipulation/scripts/manip_basic_control.py:1]: didnt get u

Comment (praveenVnktsh) [reply src/manipulation/scripts/manip_basic_control.py:1]: 3d offsets are now required due to tissue paper

File: src/manipulation/scripts/fsm.py
Comment (praveenVnktsh) [reply src/manipulation/scripts/fsm.py:1]: will fix

File: src/manipulation/scripts/visual_servoing.py
Comment (AbhinavGupta121) [inline src/manipulation/scripts/visual_servoing.py:1]: Why exit()??

File: src/manipulation/scripts/manip_basic_control.py
Comment (AbhinavGupta121) [reply src/manipulation/scripts/manip_basic_control.py:1]: goal id is just a number which is currently 47 or 4. Maybe use a dictionary with keys as strings of goals like 'banana' or 'apple' etc and map them to these goal id numbers later on if needed but It'll give more clarity to the reader

File: src/manipulation/scripts/visual_servoing.py
Comment (AbhinavGupta121) [inline src/manipulation/scripts/visual_servoing.py:1]: I assume this is same as the previous visual serving code?

Comment (praveenVnktsh) [reply src/manipulation/scripts/visual_servoing.py:1]: Test code.

Comment (praveenVnktsh) [reply src/manipulation/scripts/visual_servoing.py:1]: Yes

File: src/manipulation/scripts/manip_basic_control.py
Comment (praveenVnktsh) [reply src/manipulation/scripts/manip_basic_control.py:1]: It's being mapped internally as you suggested. However I would prefer if we don't change the service data member and retain it as integers since it will affect a lot more things beyond this subsystem.

**Review by AbhinavGupta121 — APPROVED** [review]
GGWP
